app/src/ereferrals/java/org/eyeseetea/malariacare/strategies/BaseActivityStrategy.java

```diff
@@ -178,7 +178,6 @@ public void onCreate() {
     }
 
     public void logout() {
-        AlarmPushReceiver.cancelPushAlarm(mBaseActivity);
```

Comment:
Remove this code mean that when user do logout, the push will execute for ever? I think that logout should cancel the push process